src/net_permissions.h

```diff
@@ -35,7 +35,8 @@ enum class NetPermissionFlags : uint32_t {
     // unlimited amounts of addrs.
     Addr = (1U << 7),
 
-    // True if the user did not specifically set fine grained permissions
+    // True if the user did not specifically set fine grained permissions in the
+    // -whitebind or -whitelist commands.
```

Comment:
Concept ACK, the clarification seems worth adding for future readers of the code. Suggest the following improvements to the proposal.

```diff
 
-    // True if the user did not specifically set fine grained permissions in the
-    // -whitebind or -whitelist commands.
+    // True if the user did not specifically set fine-grained permissions with
+    // the -whitebind or -whitelist configuration options.
```

Reply:
+1

Reply:
Done, thanks!